addons/im_livechat/tests/test_chatbot_internals.py

```diff
@@ -186,10 +186,12 @@ def get_forward_op_bus_params():
             )
             transfer_message_data["mail.thread"][0]["display_name"] = "Testing Bot"
             joined_message_data = Store(messages[0]).get_result()
+            partner_employee = self.partner_employee
+            partner_employee_link = f'<a href="#" data-oe-model="res.partner" data-oe-id="{partner_employee.id}">@{partner_employee.name}</a>'
```

Comment:
separate variables are not really helpful, use the content directly in body check